client/src/components/QuestionsAnswers/AnswerForm.jsx

```diff
@@ -92,18 +91,17 @@ const AnswerForm = ({ productId, productName, question }) => {
   };
 
   return (
-    <div>
-      <button onClick={ handleShow }>Add Answer</button>
+    <div className='answer-form-container'>
       <Modal
         show={ show }
-        onHide={ handleClose }
+        onHide={ handleFormClose }
         backdrop='static'
         keyboard={ false }
         centered >
         <Modal.Header closeButton>
           <Modal.Title>
             Submit Your Answer<br></br>
```

Comment:
This line reads a little strange, but I'm guessing the .Title handles the header-tag-ness of this?

Reply:
unsure what your comment means... but I think so? Modal.Title just makes the text inside that tag much larger/title-size, compared to the rest of the text in the form. Let me know if that doesn't answer your question!

Reply:
I was referring to line 105 that just says, "Submit Your Answer"<br></br>. It just reads a little weird.